music21/harmony.py

```diff
@@ -2492,6 +2493,32 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBassIndication(Harmony):
+    isFigure = True
+    tstamp = 1
+    def __init__(self, figs=None, **keywords):
+        super().__init__(**keywords)
+        if figs:
+            if isinstance(figs, list):
+                fig_string = str(figs[0])
```

Comment:
I suspect this whole block could be just:

`",".join(figs)`

You're also duplicating the first element in your string by starting your `fig_string` with the first element. Is that intentional?

Reply:
Thanks for this. Didn't see this error in the result. Seems the notation object sorts out duplicates.